deepcell/metrics.py

```diff
@@ -213,9 +215,11 @@ def __init__(self,
 
         self.merge_indices = {}
         self.merge_indices['y_true'] = []
+        self.merge_indices['y_pred'] = []
```

Comment:
I think this is a slightly cleaner way to define these dictionaries.
```suggestion
        self.merge_indices = {
            'y_true': [],
            'y_pred': []
        }
```